### PR TITLE
html-movie-seat-booking with filling seats from html

### PR DESCRIPTION
**Pull request by ses3332002**
### html-movie-seat-booking
[Demo](https://ses3332002.github.io/kottans-frontend/html-movie-seat-booking/)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/ses3332002/html-movie-seat-booking/index.html [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Homework01</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>Наши котики</h1>
+      <img class="poster" src="./img/poster.jpg" alt="постер" />
+      <p class="session_date">04.02.2020</p>
```
Comment (A-Ostrovnyy) [inline submissions/ses3332002/html-movie-seat-booking/index.html]: For date and time preferable to use `<time>` tag [Read more](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time)

File: submissions/ses3332002/html-movie-seat-booking/index.html [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+            for="seat8_8"
+            class="seat"
+          ></label>
+        </div>
+        <input type="submit" class="seats_submit" value="Заказать" />
+      </form>
+      <aside class="auditorium__description">
+        <div class="description"></div>
+        <div class="description description-vip"></div>
+        <div class="description__text">- свободно</div>
```
Comment (A-Ostrovnyy) [inline submissions/ses3332002/html-movie-seat-booking/index.html]: Logically better to group these blocks.
```suggestion
        <div class="description">
            <p class="description__text">
                <span class="description__seat"></span>
                <span class="description__seat description__seat-vip"></span>
             - свободно </p>
        </div>
```

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
Special thanks for movie.
See my comments

File: submissions/ses3332002/html-movie-seat-booking/index.html (hunk 1 above)
Comment (ses3332002) [reply submissions/ses3332002/html-movie-seat-booking/index.html]: I've remade it

File: submissions/ses3332002/html-movie-seat-booking/index.html (hunk 2 above)
Comment (ses3332002) [reply submissions/ses3332002/html-movie-seat-booking/index.html]: I've remade it

File: package.json [hunk 3]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint submissions/",
     "lint:write": "eslint --fix submissions/",
-    "prettier": "prettier --write \"submissions/**\""
```
Comment (A-Ostrovnyy) [inline package.json]: Don't change config files.)

**Review by A-Ostrovnyy — CHANGES_REQUESTED** [review]
Good work!
For future avoid editing config files.
No need to add image or other files what no needed to be reviewed in PR(remove them)

File: package.json (hunk 3 above)
Comment (ses3332002) [reply package.json]: Ok, but Anastasiya Mashoshyna asked me to do this while she was tuning repo verifications. In the beginning Prettier was tuned to check only *.js files.
I'll remove config files from PR.

Comment (ses3332002) [reply package.json]: Done, but I have some troubles with removing package.json and package-lock.json.
Git does not see this files

Comment (AMashoshyna) [reply package.json]: @ses3332002 Try this:
```
git checkout master package.json package-lock.json

```
This should bring the files back (and make them staged for commit)
Then you can commit with something like "undo file removal".

Comment (ses3332002) [reply package.json]: Thank you, but as I understand I need this files NOT to be in this PR, so the present state is exactly what I need.
Or, may be, I am wrong?
I guess the right version of package.json for main branch I can get with git pull, can't I?

Comment (A-Ostrovnyy) [reply package.json]: @ses3332002  If I merge your PR files `package.json package-lock.json` will be removed from all repositories.
Run command from @AMashoshyna 's comment.

Comment (ses3332002) [reply package.json]: Done

**Review by A-Ostrovnyy — APPROVED** [review]
Well done!

**Pull request merged**